var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -166,7 +166,7 @@ class Qt(Package):
         "https://src.fedoraproject.org/rpms/qt5-qtlocation/raw/b6d99579de9ce5802c592b512a9f644a5e4690b9/f/qtlocation-gcc10.patch",
         sha256="78c70fbd0c74031c5f0f1f5990e0b4214fc04c5073c67ce1f23863373932ec86",
         working_dir="qtlocation",
-        when="@5.15.10: %gcc@10:",
+        when="@5.14: %gcc@10:",
```

Comment:
Patch applies cleanly as far back as 5.9 (with fuzz), when the mapbox-gl-native submodule was added.
```suggestion
        when="@5.9: %gcc@10:",
```